src/applications/coronavirus-chatbot/tests/00-load-error.cypress.spec.js

```diff
@@ -0,0 +1,27 @@
+const Timeouts = require('platform/testing/e2e/timeouts');
+
+const loadError = 'div.usa-alert.usa-alert-error';
+const loadErrorHeading = 'h3.usa-alert-heading';
+const coronavirusChatbotPath = '/coronavirus-chatbot/';
+
+describe('Chatbot Load Error', () => {
+  it('Shows proper error message', () => {
+    cy.intercept('/v0/feature_toggles?&cookie_id=*', {
```

Comment:
Is `?&cookie_id=*` needed here?

Reply:
Interesting - the first time I tried this without it, it failed the request.  Trying it now without it, it seems to have worked.  Possible I had the formatting incorrect the first time.  I've removed this in both files and left just the base `/v0/feature_toggles`